plugins/org.locationtech.udig.catalog.jgrass/src/org/locationtech/udig/catalog/jgrass/core/JGrassServiceTest.java

```diff
@@ -9,97 +9,101 @@
-//            // one mapset
-//            List<IResolve> mapsetresources = service.members(null);
-//            assertNotNull(mapsetresources);
-//            assertEquals(mapsetresources.size(), 1);
-//
-//            for( IResolve mapset : mapsetresources ) {
-//
-//                // get the lsit of maps from the mapset
-//                List<IResolve> mapresources = mapset.members(null);
-//                assertNotNull(mapresources);
-//                assertEquals(mapresources.size(), 7);
-//
-//                boolean archsites = false;
-//                boolean geology = false;
-//                boolean elevationdem = false;
-//                boolean aspect = false;
-//                boolean bugsites = false;
-//                boolean roads = false;
-//                boolean fields = false;
-//                for( IResolve map : mapresources ) {
-//                    assertTrue(map instanceof JGrassMapGeoResource);
-//
-//                    IGeoResourceInfo info = ((JGrassMapGeoResource) map).getInfo(null);
-//
-//                    if ("archsites".equals(info.getName()))
-//                        archsites = true;
-//                    if ("geology".equals(info.getName()))
-//                        geology = true;
-//                    if ("elevation.dem".equals(info.getName()))
-//                        elevationdem = true;
-//                    if ("aspect".equals(info.getName()))
-//                        aspect = true;
-//                    if ("bugsites".equals(info.getName()))
-//                        bugsites = true;
-//                    if ("roads".equals(info.getName()))
-//                        roads = true;
-//                    if ("fields".equals(info.getName()))
-//                        fields = true;
-//
-//                }
-//                assertTrue(archsites);
-//                assertTrue(geology);
-//                assertTrue(elevationdem);
-//                assertTrue(aspect);
-//                assertTrue(bugsites);
-//                assertTrue(roads);
-//                assertTrue(fields);
-//
-//            }
-//
-//        } catch (IOException e) {
-//            e.printStackTrace();
-//            fail("error creating service");
-//        }
-//    }
-//}
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+
+import org.eclipse.core.runtime.FileLocator;
+import org.junit.Test;
+import org.locationtech.udig.catalog.CatalogPlugin;
+import org.locationtech.udig.catalog.IGeoResourceInfo;
+import org.locationtech.udig.catalog.IResolve;
+import org.locationtech.udig.catalog.IService;
+import org.locationtech.udig.catalog.IServiceFactory;
+import org.locationtech.udig.catalog.jgrass.JGrassPlugin;
+
+public class JGrassServiceTest {
```

Comment:
wondering why this class is in "production" bundle and not in a seperate test module... 

I guess this is the reason why the whole content was a comment :P

Reply:
I moved the JUnit test, you may have another look :)

Reply:
@sschulz92 Please have a look at changes in #531 which adresses the move of the test-case into a new bundle with test-dataset